### DISCUSSION TITLE
怎么增强PP-OCRv5_server_rec的特殊符号的识别效果，且不影响之前中文英文数字的识别效果？

### DISCUSSION THREAD

**Discussion by daichaofan0805** [PaddlePaddle/PaddleOCR#17962] [Q&A]
目前我使用了一些特殊符号的图片作为我的数据集，大概有300张，学习率设置成了0.00001，但是我训练出来的模型影响到了原本识别效果好的图片，我应该怎么调整训练数据集或者配置呢？是否应该在数据集里面加上中文的数据集图片呢？
训练前后对比
<img width="500" height="400" alt="image" src="https://github.com/user-attachments/assets/9ac4ed26-0e1d-4909-97c2-c30c370452f9" />
我的训练集如下<img width="600" height="500" alt="image" src="https://github.com/user-attachments/assets/3914d504-9e75-47b2-a0f1-b723f6b1635d" />
我的训练配置如下
Global:
  model_name: PP-OCRv5_server_rec # To use static model for inference.
  debug: false
  use_gpu: true
  epoch_num: 15
  log_smooth_window: 20
  print_batch_step: 1
  save_model_dir: D:/paddleocr/PP-OCRv5_server_rec
  save_epoch_step: 1
  eval_batch_step: [0, 10]
  cal_metric_during_train: true
  calc_epoch_interval: 1
  pretrained_model: ./pretrain_models/PP-OCRv5_server_rec_pretrained.pdparams
  checkpoints:
  save_inference_dir:
  use_visualdl: false
  infer_img: doc/imgs_words/ch/word_1.jpg
  character_dict_path: ./ppocr/utils/dict/ppocrv5_dict.txt
  max_text_length: &max_text_length 25
  infer_mode: false
  use_space_char: true
  distributed: true
  save_res_path: ./output/rec/predicts_ppocrv5.txt
  d2s_train_image_shape: [3, 32, 320]

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.00001
    warmup_epoch: 1
  regularizer:
    name: L2
    factor: 3.0e-05

Architecture:
  model_type: rec
  algorithm: SVTR_HGNet
  Transform:
  Backbone:
    name: PPHGNetV2_B4
    text_rec: True
  Head:
    name: MultiHead
    head_list:
      - CTCHead:
          Neck:
            name: svtr
            dims: 120
            depth: 2
            hidden_dims: 120
            kernel_size: [1, 3]
            use_guide: True
          Head:
            fc_decay: 0.00001
      - NRTRHead:
          nrtr_dim: 384
          max_text_length: *max_text_length

Loss:
  name: MultiLoss
  loss_config_list:
    - CTCLoss:
    - NRTRLoss:

PostProcess:  
  name: CTCLabelDecode

Metric:
  name: RecMetric
  main_indicator: acc

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./rec_examples/
    ext_op_transform_idx: 1
    label_file_list:
    - ./rec_examples/train.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - RecAug:
    - MultiLabelEncode:
        gtc_encode: NRTRLabelEncode
    - RecResizeImg:
        image_shape: [3, 32, 320]
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_gtc
        - length
  loader:
    shuffle: true
    batch_size_per_card: 32
    drop_last: true
    num_workers: 2
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./rec_examples/
    label_file_list:
    - ./rec_examples/val.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - MultiLabelEncode:
        gtc_encode: NRTRLabelEncode
    - RecResizeImg:
        image_shape: [3, 32, 320]
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_gtc
        - length
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 32
    num_workers: 2
